biosimulators_utils/sedml/data_model.py

```diff
@@ -578,6 +578,7 @@ def __init__(self, id=None, name=None, source=None, language=None, changes=None)
         self.source = source
         self.language = language
         self.changes = changes or []
+        self.structural_changes = []
```

Comment:
The data structures are chosen to mirror SED-ML. If the data structure is changed, every downstream consumer needs to be updated. We likely don't want to change this. Instead, a method could be added to return a subset of the changes.

Reply:
Would it be OK to add just a boolean 'has_structural_changes'?  I can certainly add a method to check at runtime instead, and maybe we need to if people are editing these on the fly.